pkg/envoy/admin/kds_client.go

```diff
@@ -42,124 +53,137 @@ func (k *kdsEnvoyAdminClient) PostQuit(context.Context, *core_mesh.DataplaneReso
 	panic("not implemented")
 }
 
-func (k *kdsEnvoyAdminClient) ConfigDump(ctx context.Context, proxy core_model.ResourceWithAddress) ([]byte, error) {
+type message interface {
+	grpc.ReverseUnaryMessage
+	GetError() string
+}
+
+func startTrace(ctx context.Context, tracer trace.Tracer, name string) context.Context {
+	ctx, _ = tracer.Start(
+		ctx,
+		name,
+		trace.WithSpanKind(trace.SpanKindClient),
+		// We fake attributes for the reverse unary gRPC service
```

Comment:
why do we need fake attributes?

Reply:
"Fake" is maybe the wrong word but the problem is that normally these are set by `otelgrpc`, and we have a kind of gRPC service tunneled through around service here. Perhaps I should just remove `RPCSystemGRPC`

Reply:
Removed